.github/workflows/rc.yml

```diff
@@ -32,7 +33,13 @@ jobs:
           helm-repository-base-url: ${{ env.HELM_REPO_BASE_URL }}
           helm-repository-token: ${{ secrets.BOT_GITHUB_TOKEN }}
           git-username: ${{ secrets.BOT_GITHUB_USERNAME }}
-          do-push: ${{ github.event_name == 'push' }}
```

Comment:
did you remove this on purpose?

Reply:
Yes, we don't need to have this condition for push on tag

Reply:
ah because we're in the rc workflow, ok